pyccel/codegen/printing/ccode.py

```diff
@@ -1963,6 +2014,31 @@ def _print_FunctionCall(self, expr):
         else:
             return call_code
 
+    def init_lists_or_sets(self, expr, container_var):
```

Comment:
I think this function should be broken down into two functions, one for lists and another one for sets. Either this or make it initialize all containers

Reply:
Would `init_stc_container()` be a more fitting function name?

Reply:
I don't see any benefit to splitting this given that the code is identical. I also don't think it should initialise arrays as the code would be completely different. I like the name `init_stc_container()`

Reply:
updated 749d5e524ccfb0319f0372cbe3d9cd7404eae994

Reply:
@jalalium Can we close this conversation?